packages/adblocker/src/filters/network.ts

```diff
@@ -887,6 +888,16 @@ export default class NetworkFilter implements IFilter {
             mask = setBit(mask, NETWORK_FILTER_MASK.isReplace);
             optionValue = value;
 
+            break;
+          case 'removeparam':
+            // TODO: Support regex
+            if (negation || value.startsWith('/')) {
```

Comment:
Should we also check that value is not an empty string here?

Reply:
Empty string means to remove all params from url. We should accept it here.